src/orga/subteams.rst

```diff
@@ -166,3 +166,30 @@ the following are members of the staged-recipes team and have commit rights.
  - Peter M. Landwehr <>
  - Patrick Sodré <psodre@gmail.com>
  - Sylvain Corlay <sylvain.corlay@gmail.com>
+
+
+
+Miniforge Sub-Team
+============
+
+Role
+----
+Develop and manage miniforge installers for conda-forge
+
+Charter
+-------
+Dynamic
+
+Responsibility
+--------------
+This sub-team is broadly responsible for developing, maintaining and releasing
+miniforge installers.
+
+Members
+-------
+In addition to [core](https://github.com/conda-forge/conda-forge.github.io/blob/master/src/core.csv)
```

Comment:
Could this go in the above section? Just because core has rights doesn't make them members.

Reply:
Yeah, it seems like this was Copy-pasted from the other sections, which we should also adjust but maybe that is a different PR

Reply:
Which section?

Reply:
The responsibilities section (namely the commit rights bit)

Reply:
Sure. Can you suggest a wording or push to this branch?

Reply:
Will do (pending discussion below on exact language)